jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -493,6 +493,11 @@ properties:
             type: string
             fits_keyword: LAMP
             blend_table: True
+          lamp_mode:
+            title: Operating mode for NIRSPEC internal lamp exposures
```

Comment:
For consistency with what's been implemented in JWSTKD-320 the title should be "Lamp operating mode".

Reply:
Fixed